daemon/buildinfo.c

```diff
@@ -193,7 +200,11 @@ void print_build_info(void) {
     printf("    libcap:                  %s\n", FEAT_LIBCAP);
     printf("    libcrypto:               %s\n", FEAT_CRYPTO);
     printf("    libm:                    %s\n", FEAT_LIBM);
+#if defined(ENABLE_ACLK) && !defined(ACLK_NO_LWS)
```

Comment:
Is it possible to have `defined(ENABLE_ACLK)` and `defined(ACLK_NO_LWS)` at the same time?

Shouldn't it be enough if it was just:
```
#if defined(ENABLE_ACLK)
...
#else
```

Reply:
yeah you are of course right
currently, it is not possible (considering there is no bug in configure) when `ACLK-NG` is in the picture then yes but that should be in related PR then
will drop it